.github/workflows/new-issue.yml

```diff
@@ -9,21 +9,106 @@ name: "new-issue"
 on: # any issue opened
   issues:
     types: [opened]
-
 jobs:
-  opened:
+  triage:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v2.0.0 
-    - name: issue comment
-      id: issue_comment
-      uses: ./
-      with: 
-        repo-token: ${{ secrets.GITHUB_TOKEN}}
-        message: >
-          Thank you for filing this issue.  We appreciate your feedback and will review the issue as soon as possible. 
-          Remember, however, that GitHub isn't a mechanism for receiving support under any agreement or SLA. 
-          If you require immediate assistance, contact Salesforce Customer Support.
-
-    - name: log action output
-      run: echo ${{steps.run_action.outputs.message}}     
+    - uses: github/issue-labeler@v2.0
+      with:
+        repo-token: "${{ secrets.GITHUB_TOKEN }}"
+        configuration-path: .github/labeler.yml
+        enable-versioned-regex: 0
+  missed-info:
```

Comment:
This is a lot of work and looks super messy to get the message on an issue. Instead, couldn't we just trigger when the label is added? I.e. in https://github.com/forcedotcom/cli/blob/main/.github/workflows/new-label.yml#L14 you could do something like:

```yaml
- name: missing-info
      id: issue_comment
      uses: ./
      with: 
        repo-token: ${{ secrets.GITHUB_TOKEN}}
        label: 'More Information Required'
        message: >
          Your issue is missing required information from the [bug template](https://github.com/forcedotcom/cli/blob/main/.github/ISSUE_TEMPLATE/bug_report.md). When you provide us this information we'll triage and reproduce the issue.
    - name: log action output
      run: echo ${{steps.run_action.outputs.message}}  
```

Reply:
On the quip doc, it says that we want to alert the user with the missing headers, can we talk about this later?